velox/connectors/hive/storage_adapters/gcs/GcsFileSystem.cpp

```diff
@@ -325,5 +363,20 @@ void GcsFileSystem::rmdir(std::string_view path) {
   }
 }
 
+void registerOAuthCredentialsProvider(
```

Comment:
Could we directly move this register function into the RegisterGcsFileSystem.cpp file?